### PR TITLE
W-41 Cities and countries catalog

### PR DESCRIPTION
**Pull request by alllpim**
https://trello.com/c/9ViUij0l

URL: http://localhost:3006/catalog

### CODE REVIEW

File: src/components/dialogs/CountryCatalogDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+              sx={{ mb: 3 }}
+            />
+            <Box sx={{ display: "flex" }}>
+              <Button
+                variant="contained"
+                type="submit"
+                style={{ margin: "0 0 10px 0" }}
+              >
+                Submit
+              </Button>
```
Comment (MonkeyKingBar-bit) [inline src/components/dialogs/CountryCatalogDialog.tsx:91]: Dialog doesn't close, when click on submit btn
And it doesn't add country in country table, only in console
image

File: src/pages/catalogs/countries-cities/CountriesCitiesPage.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import CountryCatalog from "../../../components/lists/CountryCatalog";
+import CityCatalog from "../../../components/lists/CityCatalog";
+import { Field, FormikProvider, useFormik } from "formik";
+import React from "react";
+import { Country } from "country-state-city";
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "../../../store";
+import { CountryData } from "../../../components/dialogs/CountryCatalogDialog.types";
+import { clientApi } from "../../../services/clientApi";
+import { clearError } from "../../../store/errorSlice";
```
Comment (MonkeyKingBar-bit) [inline src/pages/catalogs/countries-cities/CountriesCitiesPage.tsx]: Maybe need to delete unused files

**Review by MonkeyKingBar-bit — CHANGES_REQUESTED** [review]
U need to check the urls & fix adding data
image

**Review by NDragneelL9 — APPROVED** [review]

File: src/components/dialogs/CountryCatalogDialog.tsx (hunk 1 above)
Comment (alllpim) [reply src/components/dialogs/CountryCatalogDialog.tsx:91]: try to use last versions of back and front servers for testing

**Review by Lonely-fr — APPROVED** [review]

**Review by helmy-er — APPROVED** [review]

**Review by MonkeyKingBar-bit — APPROVED** [review]

**Pull request closed without merging**